{"name": "test_invalid_login[edge]", "status": "passed", "attachments": [{"name": "Screenshot for test_invalid_login[edge]", "source": "10358fec-5c77-4490-bb1e-7dd3967de45b-attachment.png", "type": "image/png"}, {"name": "log", "source": "b1f43f61-0c44-4f9d-a3ab-2d1e9938fa83-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "setup", "value": "'edge'"}], "start": 1725143896264, "stop": 1725143897478, "uuid": "e67fb941-2ec8-494f-8b89-c306241d1753", "historyId": "6322854d6259509c65287a0b2491d432", "testCaseId": "723ff608e28b98ac19ffa524f90fe2fd", "fullName": "testCases.test_login.Test_Login#test_invalid_login", "labels": [{"name": "severity", "value": "blocker"}, {"name": "parentSuite", "value": "testCases"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "Test_Login"}, {"name": "host", "value": "CNXFCM5LHW3MGRL"}, {"name": "thread", "value": "12004-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testCases.test_login"}]}